src/routes/Update.svelte

```diff
@@ -26,7 +26,8 @@
 
   async function updateHandler() {
     updating = true;
-    await installUpdate();
+    let update = await check();
+    await update?.downloadAndInstall();
```

Comment:
maybe in future, shift this updateHandler and other checkers into a update-service